src/Harpoon.cpp

```diff
@@ -13,11 +13,11 @@
 
 
 /// Creates the client that is to connect to the server/Kraken
-Harpoon::Harpoon(): 
-mQueueLength(1), //Number of allowed messages in queue
-mTimeoutMs(300000), //5 minutes
-mOffset(0),
-mChunk(nullptr) {
+Harpoon::Harpoon():
+   mQueueLength(1), //Number of allowed messages in queue
```

Comment:
sorry about the formatting. It was wrong before. If it is annoying to see it "corrected" then just add the "?w=1" to your diff to ignore the whitespace diffs.